tests/integration/pythoneditor.ts

```diff
@@ -18,7 +18,45 @@ describe('PythonEditor', () => {
     cy.visit('?token=test&reset');
   });
 
-  it('opens blank python', () => {
+  it('opens blank python from launcher', () => {
     cy.get('[title="Create a new python file"][tabindex="100"]').click();
+    // TODO: Check it opened
+  });
+
+  it('close python editor', () => {
+    cy.get('.lm-TabBar-tabCloseIcon:visible').click({ multiple: true });
+  });
+
+  it('opens blank python from new menu', () => {
+    cy.get(':nth-child(1) > .lm-MenuBar-itemLabel').click();
+    cy.get(
+      ':nth-child(2) > .lm-Menu-itemSubmenuIcon > svg > .jp-icon3 > path'
+    ).click();
+    cy.get(
+      '[data-command="pyeditor:create-new-python-file"] > .lm-Menu-itemLabel'
+    ).click();
+  });
+
+  it('check toolbar exists', () => {
+    cy.get('.elyra-PythonEditor .jp-Toolbar');
+  });
+
+  it('check save button exists and icon', () => {
+    cy.get('button[title="Save file contents"]');
+    cy.get('svg[data-icon="ui-components:save"]');
+  });
+
+  it('check run button exists and icon', () => {
+    cy.get('button[title="Run"]');
+    cy.get('svg[data-icon="ui-components:run"]');
+  });
+
+  it('check stop button exists and icon', () => {
+    cy.get('button[title="Stop"]');
+    cy.get('svg[data-icon="ui-components:stop"]');
+  });
+
+  it('check select kernel dropdown exists and has python3', () => {
+    cy.get('.elyra-PythonEditor .jp-Toolbar select > option[value=python3]');
```

Comment:
Maybe the python kernel check could be more generic.
In cases of running elyra on a remote cluster, or if someday we use a cluster on a test environment , python kernels could look like [these](https://github.com/elyra-ai/elyra/blob/master/docs/source/images/python-runner.1.gif).
I would say checking if substrings have 'python' instead?

Also I think it's a good idea to have other kernel types installed to test if this dropdown only shows python ones

Reply:
Thanks for the tip. I will work to update it.

Reply:
Just a suggestion, once you're ready with this change, you can remove the WIP so we'd merge this PR with what you already have. The list of tests in the issue is a bit long, it can be delivered into smaller PRs, that way we could be having some of them earlier

Reply:
Thanks. That makes sense to break it up into smaller PRs. I have addressed this issue now.